azurerm/data_source_arm_scheduled_time_test.go

```diff
@@ -0,0 +1,147 @@
+package azurerm
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccDataSourceAzureRMScheduledTime_Daily_Today(t *testing.T) {
+	dataSourceName := "data.azurerm_scheduled_time.test"
+
+	now := time.Now().UTC()
+	scheduletime := now.Add(time.Duration(1) * time.Hour)
+
+	config := testAccDataSourceAzureRMScheduledTime_Daily(scheduletime)
+
+	expectedTime := time.Date(scheduletime.Year(), scheduletime.Month(), scheduletime.Day(), scheduletime.Hour(), 0, 0, 0, time.UTC)
+	formattedExpectedTime := expectedTime.Format(time.RFC3339)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(dataSourceName, "hour", strconv.Itoa(now.Hour()+1)),
+					resource.TestCheckResourceAttr(dataSourceName, "minute", "0"),
+					resource.TestCheckResourceAttr(dataSourceName, "second", "0"),
```

Comment:
it'd be good for one of these tests to set the Second argument if possible?

Reply:
Done!